keyboards/wete/wete.h

```diff
@@ -0,0 +1,222 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+//////////////////////////////// ANSI ////////////////////////////////
+// ANSI layout
+// 2u backspace
+// Right handed numpad
+// 1.25-1.25-1.25 or 1.5-1-1.5 bottom row
+// 2 keys to the right of space
+#define LAYOUT_ansi_rhnp( \
+    k00, k01, k02, k03,   k04,   k05, k06, k07, k08,    k0a, k0b, k0c, k0d,    k0f, k0g, k0h, k0i,   k0j, \
+    k10, k11, k12, k13,   k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e, k1f, k1g, k1h,      k1j, \
+    k20, k21, k22, k23,   k24,   k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g,   k2i,  k2j, \
+    k30, k31, k32,        k34,    k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e, k3f,     k3i,    k3j, \
+    k40, k41, k42, k43,   k44,      k46, k47, k48, k49, k4a, k4b, k4c, k4d, k4e, k4f,     k4h,  k4i, k4j, \
+    k50,      k52,        k54,  k55,  k56,              k5a,             k5d,  k5e,        k5h, k5i, k5j \
+) { \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, XXX, k0a, k0b, k0c, k0d, XXX, k0f, k0g, k0h, k0i, k0j }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e, k1f, k1g, k1h, XXX, k1j }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g, XXX, k2i, k2j }, \
+    { k30, k31, k32, XXX, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e, k3f, XXX, XXX, k3i, k3j }, \
+    { k40, k41, k42, k43, k44, XXX, k46, k47, k48, k49, k4a, k4b, k4c, k4d, k4e, k4f, XXX, k4h, k4i, k4j }, \
+    { k50, XXX, k52, XXX, k54, k55, k56, XXX, XXX, XXX, k5a, XXX, XXX, k5d, k5e, XXX, XXX, k5h, k5i, k5j } \
+}
+
+// ANSI layout
+// 2u backspace
+// Left handed numpad
+// 1.25-1.25-1.25 or 1.5-1-1.5 bottom row
+// 2 keys to the right of space
+#define LAYOUT_ansi_lhnp( \
+    k00, k01, k02, k03,   k04,   k05, k06, k07, k08,    k0a, k0b, k0c, k0d,    k0f, k0g, k0h, k0i,   k0j, \
+    k10, k11, k12, k13,   k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e, k1f, k1g, k1h,      k1j, \
+         k21, k22, k23,   k24,   k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g,   k2i,  k2j, \
```

Comment:
All your `_lhnp` layouts in `info.json` place the object for the Numpad + key ahead of the Numpad 7/8/9. Fixing it here because it's easier to change via GitHub suggestions, and also consistent with the `_rhnp` layouts.

Basically, moving `k30` up a row in the physical arrangement.

```suggestion
    k30, k21, k22, k23,   k24,   k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g,   k2i,  k2j, \
```

Reply:
Wew. Thanks for catching this. This is actually an error in my part; it should be `k20` present rather than `k30`.